src/client/content_cao.cpp

```diff
@@ -1165,14 +1165,16 @@ void GenericCAO::step(float dtime, ClientEnvironment *env)
 			m_step_distance_counter = 0.0f;
 			if (!m_is_local_player && m_prop.makes_footstep_sound) {
 				const NodeDefManager *ndef = m_client->ndef();
-				v3s16 p = floatToInt(getPosition() +
-					v3f(0.0f, (m_prop.collisionbox.MinEdge.Y - 0.5f) * BS, 0.0f), BS);
-				MapNode n = m_env->getMap().getNode(p);
+				v3f foot_pos = getPosition() * (1.0f/BS)
```

Comment:
rename to bottom pos, as not everyhas has feet

Reply:
But it's *foot*step sounds. :foot:

Reply:
@nerzhul Do you insist on this?